x/posts/internal/types/msgs.go

```diff
@@ -179,7 +179,7 @@ func (msg MsgUnlikePost) Type() string { return ActionUnlikePost }
 // ValidateBasic runs stateless checks on the message
 func (msg MsgUnlikePost) ValidateBasic() sdk.Error {
 	if !msg.PostID.Valid() {
-		return sdk.ErrUnknownRequest("Invalid post id")
+		return sdk.ErrUnknownRequest("Invalid post newID")
```

Comment:
I think this is a refactoring error

```suggestion
		return sdk.ErrUnknownRequest("Invalid post id")
```